charts/cromiam/templates/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}-ingress
+  labels:
+{{ include "cromiam.labels" . | indent 4 }}
+  annotations:
+    networking.gke.io/v1beta1.FrontendConfig: "{{ .Chart.Name }}-ingress-frontendconfig"
+    kubernetes.io/ingress.global-static-ip-name: {{ required "ingress.staticIpName value is required" .Values.ingress.staticIpName | quote }}
+{{- if not (empty .Values.ingress.preSharedCerts) }}
+    ingress.gcp.kubernetes.io/pre-shared-cert: {{ .Values.ingress.preSharedCerts | join "," | quote }}
+{{- end }}
```

Comment:
If you aren't including the logic to pull the  certs from vault also then I don't think this conditional is necessary. Not a big deal though 😄

Reply:
Don't want to supply an empty string to the annotation and get potentially undefined behavior 🤷‍♀️

Reply:
not trying to bike shed but would it be better to use the `required` function with the value in that case?